examples/bigcommerce/components/cart.tsx

```diff
@@ -21,19 +23,26 @@ type CartLineItemProps = {
 
 function CartLineItem({ lineItem }: CartLineItemProps) {
   const { updateItem, deleteItem } = useCart()
+  const product = useProduct(lineItem.product_id)
   const [cartState, setCartItemState] = useState<CartLineItemState>('initial')
+  const { t } = useTranslation('cart')
+
   return (
     <Link href={`/product/${lineItem.product_id}`}>
       <a className={`flex w-full space-x-4 items-end`} key={lineItem.id}>
-        <img src={lineItem.image_url} alt={`Product image for ${lineItem.name}`} width={70} />
+        <img
+          src={lineItem.image_url}
+          alt={`Product image for ${product.name ?? lineItem.name}`}
+          width={70}
+        />
         <div className="flex flex-col flex-grow items-start justify-between">
-          <div className="text-base text-black">{lineItem.name}</div>
+          <div className="text-base text-black">{product.name ?? lineItem.name}</div>
```

Comment:
There is no api for getting the Metafields of a lineitem, so I am preferring the `product.name` to the `lineItem.name` here.